### PR TITLE
Additional styling, dummy data for tests, search functionality, simple error handling

### PR DESCRIPTION
**Pull request by Atr**

### CODE REVIEW

File: client/src/app.jsx [hunk 1]
```diff
@@ -1,77 +1,126 @@
...
-      <OuterWrapper>
-        <InnerWrapper>
-          <ContainerAverageStars averageStarsObj={this.state.averageStarsObj} />
-          <ContainerReviews reviews={this.state.reviews} />
-        </InnerWrapper>
-      </OuterWrapper>
+      <AppOuterWrapper>
+        {this.state.err404 ?
+          (
+            <BasicTextFormatSpan>
```
Comment (ivan-cordoba) [inline client/src/app.jsx:106]: I like how you are handling listings outside of outside of 1-100

File: client/src/components/ReviewEntry.jsx [hunk 2]
```diff
@@ -2,18 +2,18 @@ import React from 'react';
...
+  ReviewTextWrapper,
 } from './styles/ReviewEntry.styles';
 
 const ReviewEntry = props => (
-  <Wrapper>
-    <ReviewUser> {props.review.first_name} </ReviewUser>
-    <ReviewDate> {props.review.review_date} </ReviewDate>
-    <ReviewText> {props.review.review_text} </ReviewText>
-  </Wrapper>
+  <ReviewEntryWrapper>
```
Comment (ivan-cordoba) [inline client/src/components/ReviewEntry.jsx:12]: These new styled component names are much more descriptive

File: client/src/app.jsx [hunk 3]
```diff
@@ -1,77 +1,126 @@
...
+      currentSearch: '',
+      err404: false,
     };
   }
 
   componentDidMount() {
-    // Using parameter 1 for now, just for testing
-    this.getListingReviews(14);
-    this.getListingAverageStars(14);
+    this.getListingReviews(this.getListingId());
```
Comment (ivan-cordoba) [inline client/src/app.jsx:41]: You are calculating the id twice here, you can refactor this to only run getListingId() once or maybe even include it in the componentDidMount() and get rid of the function entirely.

**Review by ivan-cordoba — COMMENTED** [review]
Your code looks good, I think you have a good amount of tests as well. Everything is very well organized and I was able to follow the flow of your app very easily. Good job.

**Review by ivan-cordoba — APPROVED** [review]

File: client/src/app.jsx (hunk 1 above)
Comment (Atr) [reply client/src/app.jsx:106]: Thank you !

File: client/src/components/ReviewEntry.jsx (hunk 2 above)
Comment (Atr) [reply client/src/components/ReviewEntry.jsx:12]: Thank you !  Longer for sure..

File: client/src/app.jsx (hunk 3 above)
Comment (Atr) [reply client/src/app.jsx:41]: Great call!  I will make that change.

**Pull request merged**